File: Week 6/mario.py
from cs50 import get_int

while True:
    x = get_int("Height: ")
    if x > 0 and x <= 8:
        break

for i in range(x):
    print(" " * (x - (i + 1)), end = "")
    print("#" * (i + 1), end = "")
    print("  ", end = "")
    print("#" * (i + 1), end = "")
    print()

File: Week 1/mario.c
#include <stdio.h>
#include <cs50.h>

int main(void)
{
    int n;
    do
    {
        n = get_int("Height: ");    // we get input
    }
    while (n < 1 || n > 8);    // the space that we didn't want

    for (int i = 0; i < n; i++)
    {
        for (int x = 0; n - (i + 1) > x; x++)   // spaces should repeat (n - i) times
        {
            printf(" ");
        }
        for (int j = 0; j <= i; j++)  //blocks
        {
            printf("#");
        }
        printf("\n");
    }
}















